### ISSUE TITLE
On  bsd it must we should also add as path "/usr/local/etc/fonts/fonts.conf"

### ISSUE THREAD

**Issue by rubik83** [RazrFalcon/fontdb#75]
On bsd my configuration file is only in
`/usr/local/etc/fonts/fonts.conf`
and not
`/etc/fonts/fonts.conf`

I have to add a symbolic link for it to work

**Comment by ghost** [comment]
i have the same issue on freebsd and the symlink worked.
would still be nice to have in the lib as is

**Comment by jbeich** [comment]
Duplicate of #61? Easy to patch downstream e.g., https://github.com/freebsd/freebsd-ports/commit/a2394dacc23a

**Comment by ghost** [comment]
i would try making a patch but i dont know how to use freebsd's prefixes

**Comment by MoSal** [comment]
Where do the font files themselves reside?
If they are under `/usr/local/share/fonts/`, then they should be found even without finding/reading fontconfig configuration file(s) after #71, no?

**Comment by jbeich** [comment]
cosmic-text still [uses 0.16](https://github.com/pop-os/cosmic-text/blob/1f4065c1c339/Cargo.toml#L15) series which lacks #71.

**Comment by ghost** [comment]
wouldnt it still make sense to patch in a the normal way to find the fonts.conf file for other os's?
